### ISSUE TITLE
Help ?

### ISSUE THREAD

**Issue by krzm** [bilal-fazlani/commanddotnet#377]
public class PipedInputTests
{
    [Test]
    public void Add_Given2Numbers_Should_OutputSum()
    {
        var result = new AppRunner<Calculator>().RunInMem("Add 40 20");
        result.ExitCode.Should().Be(0);
        result.Console.Out.Should().Be(@"60");
    }
}

I wanted this from docs to run.
Cant find this methods Should
Is docs outdated ?
Didnt see this method in repo.

**Comment by krzm** [comment]
Ups, found Shoud in repo.
But still, are docs outdated ?

**Comment by krzm** [comment]
Ok cloned repo, found i need nuget FluentAssertions
Shame docs dont mentions it
Old chunck of coal like me didnt know..

**Comment by drewburlingame** [comment]
Hi @krzm is this the kind of indicator that would have helped?

![image](https://user-images.githubusercontent.com/103345/141369101-ae880c6a-9529-44e6-957f-871d8fb2ca48.png)

**Comment by drewburlingame** [comment]
@krzm, other than the Should red-herring, how's your experience of the framework.  I see you've built a few console apps and possibly a similar console framework.  I'd be interested to hear your impressions..

**Comment by krzm** [comment]
Hi @drewburlingame, yes it would help. 
Problem is, test failed for unknow to me as yet reason.
I begun to like Console recently. 
I did some simple lib solutions of my own, witch are tailored to my simple needs.
I seen this vid: [https://youtu.be/dKBuvdGsYYc](url)
I thouth maby i dont need my own libs.
I started to look at CommandDotNet.
I think it's impresive.
Started in doc section getting started.
Hope to see how i can use it by doing some examples, from docs or repo.
